### DISCUSSION TITLE
Richasy.exe是什么啊

### DISCUSSION THREAD

**Discussion by pxxcjp** [Richasy/Bili.Uwp#412] [问答]
如题，这个exe每次启动都会访问我当前用户文件夹

**Comment by pxxcjp** [comment]
@Richasy

**Reply by Richasy** [reply]
我没找到这个可执行文件，哔哩也不包含访问用户文件夹的操作。你能发一下详细的检查步骤么？

**Reply by hez2010** [reply]
> 我没找到这个可执行文件，哔哩也不包含访问用户文件夹的操作。你能发一下详细的检查步骤么？

自己构建的 Bili.Uwp 的话 exe 名称就是 Richasy.exe
